climada/hazard/plot.py

```diff
@@ -41,6 +46,15 @@ def plot_rp_intensity(
         adapt_fontsize=True,
         **kwargs,
     ):
+        """
+        This function is deprecated,
+        use Impact.local_exceedance_impact and util.plot.plot_from_gdf instead.
+        """
+        LOGGER.warning(
+            "The use of Hazard.plot_rp_intensity is deprecated."
+            "Use Hazard.local_exceedance_intensity and util.plot.plot_from_gdf instead."
+        )
```

Comment:
I'd suggest to use the `@deprecated` decorator instead, like e.g., in the `climada.hazard.centr` module.
I know, atm, the climada codebase is heterogeneous in this regard but in the long run the decorator is meant to take over.

Reply:
thank you, I didn't know about this. I changed this to decorators now